consensus/istanbul/config_test.go

```diff
@@ -51,8 +52,8 @@ func TestGetConfig(t *testing.T) {
 		t.Errorf("error default config:\nexpected: %v\n", DefaultConfig)
 	}
 
-	config := DefaultConfig
-	config.Transitions = []params.Transition{{
+	config := &DefaultConfig
+	(*config).Transitions = []params.Transition{{
```

Comment:
why? DefaultConfig is already a pointer.
Also, The problem with pointer is that Go is copying the address not the object.

1. having the pointer doesn't copy the value, so DefaultConfig is modified for me.
2. you need the (*...) because you have a pointer of the pointer

Reply:
Since the `DefaultConfig` is already a pointer, I am trying to copy the object not the pointer; that's why i used `config := &DefaultConfig`. 
To change the `Transitions`, I used the `(*config).Transitions`. let me know if i did something wrong